style.css

```diff
@@ -711,6 +711,14 @@ textarea.message::placeholder {
   margin-left: 2vw;
 }
 
+.error-message {
```

Comment:
- [OPTIONAL] I suggest before merging the project raise the font size of the error message or put it after the button because right now you can't see it properly and after the changes, it will look perfect. Keep shining 💪